app/styles/components/c-box-triangle.scss

```diff
@@ -1,31 +1,31 @@
-@import '../base/text.scss';
-@import '../base/colors.scss';
+@import '../_settings.scss';
 
-.c_box_triangle {
-  margin: 0 auto;
-  width: 52px;
-  height: 52px;
-  transform: rotate(-135deg);
-  -webkit-transform: rotate(-135deg);
+.c-box-triangle {
   -moz-transform: rotate(-135deg);
-  -o-transform: rotate(-135deg);
   -ms-transform: rotate(-135deg);
-  border-right: 2px solid $color9;
-  border-bottom: 2px solid $color9;
-  margin-bottom: -100px;
+  -o-transform: rotate(-135deg);
```

Comment:
Useless: remove all the prefixes, `autoprefixer` will do it for us.

`transform: rotate(-135deg);`